### ISSUE TITLE
avoid close the keyboard while I request focus to other TextArea or TextField Component

### ISSUE THREAD

**Issue by DurankGts** [codenameone/CodenameOne#3157]
Please help me.

    private void editNextDigit1(TextArea editing, TextArea nextEditing) {

        editing.stopEditing(new Runnable() {
            @Override
            public void run() {
                nextEditing.startEditingAsync();
            }
        });
    }

**Comment by DurankGts** [comment]
In check in the android port source code that you are forcing to close the Keyboard.

![image](https://user-images.githubusercontent.com/16245755/87082011-50362f80-c1f8-11ea-9b74-8404d90c0053.png)

here

![image](https://user-images.githubusercontent.com/16245755/87082054-6512c300-c1f8-11ea-97e1-382010705690.png)

here
![image](https://user-images.githubusercontent.com/16245755/87084422-34348d00-c1fc-11ea-9b5c-a8e398697aa9.png)

**Comment by jsfan3** [comment]
Yes, the current behaviour is also confirmed by this Shai's answer: https://stackoverflow.com/a/59816111/1277576
In the past, I workarounded this issue as in the last comment of that linked answer, but it's not a proper solution and I don't recommend it. I personally no longer use that workaround. A proper solution is desirable.

**Comment by DurankGts** [comment]
If I have 5 TextArea of Text field component, the actual behaivor of the keyboard is close and open in every call to editing the next TextArea or TextField component. the Behaivor of Keep the keyboard open is more convenient in this case.

**Comment by jsfan3** [comment]
Does it also happen by pressing the "next" key on the virtual keyboard?

**Comment by DurankGts** [comment]
yes
